client/app/components/Comment.jsx

```diff
@@ -1,13 +1,16 @@
 import React, { PropTypes } from 'react';
 import marked from 'marked';
 
-const Comment = React.createClass({
-  displayName: 'Comment',
-
-  propTypes: {
+class Comment extends React.Component {
+  constructor(props) {
```

Comment:
I think it's better to avoid using `constructor` in React components, especially since we aren't even setting `state` here. If we do need to set state, it should be via class a class property. For example, if we want to have a state of `{author: "Timmy", text: "hello world"}, you just do this:

```javascript
class Comment extends React.Component {
  author = "Timmy";
  text = "hello world";
....
```